src/com/intellij/idea/plugin/hybris/impex/file/actions/CopyImpexFileAction.java

```diff
@@ -0,0 +1,54 @@
+package com.intellij.idea.plugin.hybris.impex.file.actions;
+
+import com.intellij.ide.projectView.impl.nodes.PsiFileNode;
+import com.intellij.idea.plugin.hybris.actions.AbstractCopyFileToHybrisConsoleAction;
+import com.intellij.idea.plugin.hybris.actions.ActionUtils;
+import com.intellij.idea.plugin.hybris.tools.remote.console.HybrisConsole;
+import com.intellij.idea.plugin.hybris.tools.remote.console.view.HybrisConsolePanel;
+import com.intellij.idea.plugin.hybris.toolwindow.CopyFileToHybrisConsoleDialog;
+import com.intellij.openapi.actionSystem.AnActionEvent;
+import com.intellij.openapi.actionSystem.DataContext;
+import com.intellij.openapi.project.Project;
+import org.jetbrains.annotations.NotNull;
+
+import javax.swing.tree.DefaultMutableTreeNode;
+import javax.swing.tree.TreePath;
+
+public class CopyImpexFileAction extends AbstractCopyFileToHybrisConsoleAction {
+
+    private static final String IMPEX_CONSOLE_TITLE = "Hybris Impex Console";
+
+    @Override
+    public void update(@NotNull final AnActionEvent e) {
+        final DataContext dataContext = e.getDataContext();
+        e.getPresentation()
+         .setEnabledAndVisible(ActionUtils.isHybrisContext(dataContext) && isRequiredFileExtension(
+             dataContext, ".impex", false));
+    }
+
+    @Override
+    public void actionPerformed(@NotNull final AnActionEvent e) {
```

Comment:
Seems that common code could be also moved to AbstractCopyFileToHybrisConsoleAction